### PR TITLE
grid_printer and list_lab

### PR DESCRIPTION
**Pull request by nichx313**
This PR is to submit the grid_printer.py and list_lab.py files. I believe the assignments have been closed in Open edX, if my prior submissions for those 2 could be dropped or updated, that would be great. If they need deleted, let me know and I'll resubmit with this PR link.

I've done work on other files/there are other commits, but those are not yet ready for review/submission. I've been pushing all my work to my fork as I'm working between multiple computers and I'm trying to keep files up to date across them.

### CODE REVIEW

File: students/nick_miller/lesson02/grid_printer.py [hunk 1]
```diff
@@ -27,10 +45,60 @@ def grid_printer():
...
+    minus = "- "
+    pipe = "| "
+    space = " "
+    horiz = h * (plus + (w * minus)) + plus + "\n"
+    vert = w * (h * (pipe + (w * (2 * space))) + pipe + "\n")
+    stack = h * (horiz + vert) + horiz
+    print(stack)
+
+
+print_grid2(5, 2)
```
Comment (charlesrobison) [inline students/nick_miller/lesson02/grid_printer.py]: Works well!

File: students/nick_miller/lesson03/list_lab.py [hunk 2]
```diff
@@ -30,38 +30,41 @@
 # Ask the user for a number and display the number back to the user and the fruit corresponding to that number
 # (on a 1-is-first basis). Remember that Python uses zero-based indexing, so you will need to correct.
 
+pick = str(len(fruits))
```
Comment (charlesrobison) [inline students/nick_miller/lesson03/list_lab.py]: Good!

File: students/nick_miller/lesson03/list_lab.py [hunk 3]
```diff
@@ -92,50 +95,98 @@
...
+for fruit in fruits:
+    question = "Do you like " + fruit + "? (y/n): "
+    yesorno = input(question)
+    drop = fruits.index(fruit)
+    while yesorno != "y" and yesorno != "n":
+        yesorno = input("Please enter y or n: ").lower()
+    if yesorno == "n":
+        delfruits.append(drop)
+for i in delfruits:
+    fruits.pop(i)
```
Comment (charlesrobison) [inline students/nick_miller/lesson03/list_lab.py]: Getting an error here

File: students/nick_miller/lesson03/mailroom.py [hunk 4]
```diff
@@ -43,26 +168,13 @@ def report(db=donor_db):
...
-    print()
+    print("-" * 76)
+    for row in report_don:
+        print("{:<18s}   {:>15.2f}   {:12d}   {:>20.2f}".format(*row))
 
 
 def quit_prog():
-    print("quitting - see you next time")
+    print("Quitting - See you next time.")
     quit()
```
Comment (charlesrobison) [inline students/nick_miller/lesson03/mailroom.py]: This isn't actually quit the program - is it finished?

Comment (nichx313) [reply students/nick_miller/lesson03/mailroom.py]: Updated to import sys and properly quit. Will re-submit on next PR. Thanks!

**Comment by natasha-aleksandrova** [comment]
@charlesrobison looks like @nichx313 addressed some of the issues here.

can you please take another look?

**Comment by charlesrobison** [comment]
Update looks good!  Ready to merge

**Pull request merged**